rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -705,4 +705,50 @@ private Optional<List<BtcECKey>> getGenesisFederationPublicKeys() {
     public long getGasEstimationCap() {
         return configFromFiles.getLong(PROPERTY_RPC_GAS_ESTIMATION_CAP);
     }
+
+    public DbKind databaseKind() {
+        return DbKind.ofName(configFromFiles.getString("keyvalue.datasource"));
+    }
+
+    public DbKind getDbKindValueFromDbKindLogFile() {
+        try {
+            File file = new File(databaseDir(), "dbKind.properties");
```

Comment:
could we extract this file name to constants?

Reply:
Can you elaborate?